### PR TITLE
Updated aks.md and added helm.md

### LINKED ISSUE #9055: [Docs] Update AKS Documentation for Service-level Port Forwarding
Issue Description:
The current documentation for AKS (Azure Kubernetes Service) in the Supported providers section contains instructions for port-forwarding that involve a complex process of identifying and selecting a specific pod. This process can be streamlined and improved to enhance user-friendliness.

Proposed Change:

Introduction of Service-level Port Forwarding: The documentation should be updated to include clear instructions on how to perform service-level port forwarding. This change will simplify the process for users and eliminate the need to manually identify and select individual pods.

### Current State
export POD_NAME=$(kubectl get pods --namespace meshery -l "app.kubernetes.io/name=meshery,app.kubernetes.io/instance=meshery" -o jsonpath="{.items[0].metadata.name}")
kubectl --namespace meshery port-forward $POD_NAME 9081:8080

### Desired State
kubectl port-forward svc/meshery --namespace meshery  9081:9081

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by aboobakersiddiqr63**
**Notes for Reviewers**
- Updated aks.md & added helm.md

This PR fixes #9055 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ✓] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit a01cffbe548b1984b204ff4184aa2330b9266666 at: https://652f5f5cab4af35406a621a7--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 5d45b2df8dd5cf803ffb3fc578e65af7d3a94f21 at: https://652fef46a49f014e0182b5c4--meshery-docs-preview.netlify.app

File: docs/pages/installation/helm.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+<div class="prereqs"><p><strong style="font-size: 20px;">Prerequisites</strong> </p> 
+<ol>
+<li><a href="https://helm.sh/docs/intro/install/" class="meshery-light">Helm</a> should be installed on your local machine.</li>
+<li>You should have access to the cluster/platform where you want to deploy Meshery.</li>
+<li>Ensure that the kubeconfig file has the correct current context/cluster configuration.</li>
+</ol>
+</div> 
+
+## Install Meshery on Your Kubernetes Cluster Using Helm V3
```
Comment (leecalcote) [inline docs/pages/installation/helm.md:19]:
```suggestion
## Install Meshery on Your Kubernetes Cluster Using Helm
```

File: docs/pages/installation/aks.md [hunk 2]
```diff
@@ -8,70 +8,84 @@ language: en
...
+
 ### Access Meshery UI
 
-Access Meshery UI by exposing it as a Kubernetes service or by port forwarding to Meshery UI.
+To access Meshery's UI, please refer to the [instruction](/reference/mesheryctl/system/dashboard) for detailed guidance.
 
-#### [Optional] Port Forward to Meshery UI
+Note: _You can also verify health of your system using [mesheryctl system check](/reference/mesheryctl/system/check)_
 
-{% capture code_content %}export POD_NAME=$(kubectl get pods --namespace meshery -l "app.kubernetes.io/name=meshery,app.kubernetes.io/instance=meshery" -o jsonpath="{.items[0].metadata.name}")
```
Comment (leecalcote) [inline docs/pages/installation/aks.md:72]: Where else is this information being captured?

**Comment by l5io** [comment]
🚀 Preview for commit 9784b3a31111394ee4c376a574429bad7b3d0d97 at: https://65307fbfb2933839e19e51fe--meshery-docs-preview.netlify.app

File: docs/pages/installation/aks.md (hunk 2 above)
Comment (leecalcote) [reply docs/pages/installation/aks.md:72]: I'm copying and capturing in the central Kubernetes install page.

Comment (leecalcote) [reply docs/pages/installation/aks.md:72]: We might want to delete, though, once the dust has settled.

File: docs/pages/installation/helm.md
Comment (leecalcote) [inline docs/pages/installation/helm.md:1]: Good work getting a page going specifically for Helm. 👍

**Review by leecalcote — APPROVED** [review]

**Pull request merged**